src/node.hpp

```diff
@@ -13,8 +13,8 @@
 #include <vector>
 #include <sys/stat.h>
 
-/* Function prototype */
-void create_log_dir();
+/* Create log dir if not exists */
+bool create_log_dir();
```

Comment:
get rid of this, we will now use static function

Reply:
Done